attestation-agent/attester/src/lib.rs

```diff
@@ -78,10 +78,6 @@ pub trait Attester {
 
 // Detect which TEE platform the KBC running environment is.
 pub fn detect_tee_type() -> Option<Tee> {
```

Comment:
Maybe we should also change the result to `Tee` rather than `Option` as at least `sample` will be returned.

Reply:
Got rid of the option. I kept the interface the same at the level of the EvidenceProvider, but adjusted things up to that.